servatrice/migrations/servatrice_0022_to_0023.sql

```diff
@@ -0,0 +1,6 @@
+-- Servatrice db migration from version 22 to version 23
+
+alter table cockatrice_rooms modify column permissionlevel enum('NONE','REGISTERED','MODERATOR','ADMINISTRATOR');
```

Comment:
Yeah they should be ' , not sure how one line came out one way and the other another.  I will fix this.